politeiawww/api/cms/v1/v1.go

```diff
@@ -193,15 +195,18 @@ type GeneratePayoutsReply struct {
 
 // Payout contains an address and an amount to be paid
 type Payout struct {
-	ContractorName string `json:"contractorname"`
-	ContractorRate uint   `json:"contractorrate"`
-	Username       string `json:"username"`
-	Month          uint   `json:"month"`
-	Year           uint   `json:"year"`
-	Token          string `json:"token"`
-	Address        string `json:"address"`
-	LaborTotal     uint   `json:"labortotal"`
-	ExpenseTotal   uint   `json:"expensetotal"`
+	ContractorName string         `json:"contractorname"`
+	ContractorRate uint           `json:"contractorrate"`
+	Username       string         `json:"username"`
+	Month          uint           `json:"month"`
+	Year           uint           `json:"year"`
+	Token          string         `json:"token"`
+	Address        string         `json:"address"`
+	LaborTotal     uint           `json:"labortotal"`
```

Comment:
These fields that are either durations or money need comments saying what unit they are (cents, minutes, etc).